File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        unordered_map<char,int> mpp;
        for(auto it:s){
            mpp[it]++;
        }
        
        for(auto ti:t){
            mpp[ti]--;
        }
        
        for(auto x:mpp){
            if(x.second!=0){
                return false;
            }
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        countS = {}
        countT = {}
        
        for i in list(s):
            if i in countS:
                countS[i] += 1
            else:
                countS[i] = 1
                
        for j in list(t):
            if j in countT:
                countT[j] += 1
            else:
                countT[j] = 1
                
        return countS==countT
            
        